dynamodb/src/main/scala/akka/stream/alpakka/dynamodb/javadsl/DynamoDb.scala

```diff
@@ -27,6 +29,22 @@ object DynamoDb {
                                                            parallelism: Int): Flow[In, Out, NotUsed] =
     scaladsl.DynamoDb.flow(parallelism)(client, operation).asJava
 
+  /**
+   * Create a Flow that emits a response for every request to DynamoDB.
```

Comment:
Add something along the lines "Intended to be used with Akka retry flows which expect the Scala `Try` type"